2021_RTCSA_dynfed/src/main.rs

```diff
@@ -31,31 +37,83 @@ struct ArgParser {
     output_dir_path: String,
 }
 
+pub fn adjust_dag_set(dag_set: &mut [Graph<NodeData, i32>]) {
```

Comment:
To clarify the process, how about the name `adjust_to_implicit_deadline` or `ensure_implicit_deadline`?
In addition, this process should be moved to util.rs and made a shared library.

Reply:
`adjust_dag_set` -> `adjust_to_implicit_deadline`
`main.rs` -> `util.rs`